tools/SetupFlow/DevHome.SetupFlow/Views/LoadingView.xaml

```diff
@@ -135,48 +116,59 @@
-                            <Grid.Resources>
-                                <ResourceDictionary>
-                                    <Style x:Key="PrimaryTextStyle" TargetType="TextBlock">
-                                        <Setter Property="Foreground">
-                                            <Setter.Value>
-                                                <SolidColorBrush Color="{ThemeResource TextFillColorPrimary}"/>
-                                            </Setter.Value>
-                                        </Setter>
-                                    </Style>
-                                    <Style x:Key="SecondaryTextStyle" TargetType="TextBlock">
-                                        <Setter Property="Foreground">
-                                            <Setter.Value>
-                                                <SolidColorBrush Color="{ThemeResource TextFillColorSecondary}"/>
-                                            </Setter.Value>
-                                        </Setter>
-                                    </Style>
-                                </ResourceDictionary>
-                            </Grid.Resources>
-                            <ImageIcon
-                                Grid.Column="0"
-                                VerticalAlignment="Top"
-                                Visibility="{x:Bind ShouldShowStatusSymbolIcon, Mode=OneWay}"
-                                Source="{x:Bind StatusSymbolIcon, Mode=OneWay}"/>
-                            <ProgressRing
-                                Grid.Column="0"
-                                Height="18"
-                                Width="18"
-                                IsActive="{x:Bind ShouldShowProgressRing, Mode=OneWay}"
-                                Visibility="{x:Bind ShouldShowProgressRing, Mode=OneWay}"/>
-                            <TextBlock Grid.Column="1" Text="{x:Bind MessageToShow, Mode=OneWay}" VerticalAlignment="Center" Style="{x:Bind ShouldShowProgressRing, Mode=OneWay, Converter={StaticResource BoolToTextBrush}}" TextWrapping="Wrap" IsTextSelectionEnabled="True"/>
-                        </Grid>
-                    </DataTemplate>
-                </ListView.ItemTemplate>
-            </ListView>
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition Height="Auto"/>
+                </Grid.RowDefinitions>
+                <ListView x:Name="NonExecutingTasks"  Grid.Row="0" ItemsSource="{x:Bind ViewModel.NonExecutingMessages, Mode=OneWay}" SelectionMode="Single">
+                    <ListView.ItemTemplate>
+                        <DataTemplate x:DataType="viewmodels:LoadingMessageViewModel">
+                            <Grid ColumnSpacing="5">
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="18"/>

```

Comment:
Is it better to set the FontSize on the ImageIcon (if that's a thing) and make this "auto"? (actually asking)

Reply:
ImageIcon does not have FontSize.  It does have Width and Height.  I tried this,  No change visually.

With a change this small, I don't think it matters?  Maybe?

But, hardcoding a width does signify to future developers "This column is supposed to stay this small"